src/shared/navgen/nav.cpp

```diff
@@ -415,8 +428,17 @@ void NavmeshGenerator::LoadTris( std::vector<float> &verts, std::vector<int> &tr
 	for ( int k = model->firstSurface, n = 0; n < model->numSurfaces; k++, n++ )
 	{
 		const dsurface_t *surface = &bspSurfaces[ k ];
+		const dshader_t *shader = &bspShaders[surface->shaderNum];
+
+		/* Patches don't have any content but their shader may
+		have contentparm. */
+		if ( SkipContent( brushShader ) )
```

Comment:
The first commit would not compile as it uses an out-of-scope variable `brushShader`.

Reply:
Ah right, it should be named `shader` and this change went into the second patch. I'm fixing this.